src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/VerificationComponents/VerifyGitopsEntities.tsx

```diff
@@ -0,0 +1,199 @@
+    return {
+      icon: 'tick-circle',
+      iconProps: { padding: { right: 'medium' }, color: Color.GREEN_700 }
+    }
+  }, [])
+  const failIconProps = React.useMemo((): { icon: IconName; iconProps: Omit<IconProps, 'name'> } => {
+    return {
+      icon: 'danger-icon',
+      iconProps: { padding: { right: 'medium' }, color: Color.ERROR, className: css.redIcon }
+    }
+  }, [])
+
+  const loadingIconProps = React.useMemo((): { icon: IconName; iconProps: Omit<IconProps, 'name'> } => {
+    return {
+      icon: 'steps-spinner',
+      iconProps: { padding: { right: 'medium' } }
+    }
+  }, [])
+
+  useEffect(() => {
+    appData?.name &&
+      trackEvent(ONBOARDING_INTERACTIONS.APP_VERIFICATION_SUCCESS, getBranchingProps(stepsProgress, getString))
+    repoData?.repository &&
+      trackEvent(ONBOARDING_INTERACTIONS.REPO_VERIFICATION_SUCCESS, getBranchingProps(stepsProgress, getString))
+    clusterData?.cluster &&
+      trackEvent(ONBOARDING_INTERACTIONS.CLUSTER_VERIFICATION_SUCCESS, getBranchingProps(stepsProgress, getString))
+  }, [appData, clusterData, repoData])
+
+  useEffect(() => {
+    trackEvent(ONBOARDING_INTERACTIONS.APP_VERIFICATION_START, getBranchingProps(stepsProgress, getString))
+    trackEvent(ONBOARDING_INTERACTIONS.REPO_VERIFICATION_START, getBranchingProps(stepsProgress, getString))
+    trackEvent(ONBOARDING_INTERACTIONS.CLUSTER_VERIFICATION_START, getBranchingProps(stepsProgress, getString))
+  }, [])
+
+  useEffect(() => {
+    const allChecksPassed = [appFailed, repoFailed, clusterFailed].every(erorr => erorr === null)
+    const isEntityLoading = [applicationLoading, repoLoading, clusterLoading].includes(true)
+    if (allChecksPassed && !isEntityLoading && !setupState?.gitopsEntitiesVerified) {
+      saveProgress(CDOnboardingSteps.DEPLOYMENT_STEPS, {
+        gitopsEntitiesVerified: true,
+        pipelineVerified: false
+      } as PipelineSetupState)
+    } else if (!allChecksPassed && !isEntityLoading && setupState?.gitopsEntitiesVerified) {
+      saveProgress(CDOnboardingSteps.DEPLOYMENT_STEPS, {
+        gitopsEntitiesVerified: false,
+        pipelineVerified: false
+      } as PipelineSetupState)
+    }
+
+    appFailed && trackEvent(ONBOARDING_INTERACTIONS.APP_VERIFICATION_FAIL, getBranchingProps(stepsProgress, getString))
+    repoFailed &&
+      trackEvent(ONBOARDING_INTERACTIONS.REPO_VERIFICATION_FAIL, getBranchingProps(stepsProgress, getString))
+    clusterFailed &&
+      trackEvent(ONBOARDING_INTERACTIONS.CLUSTER_VERIFICATION_FAIL, getBranchingProps(stepsProgress, getString))
+  }, [
+    appFailed,
+    repoFailed,
+    clusterFailed,
+    applicationLoading,
+    repoLoading,
+    clusterLoading,
+    setupState?.gitopsEntitiesVerified
+  ])
+
+  const retryVerification = (): void => {
+    clusterFailed && refetchCluster()
+    appFailed && refetchApp()
+    repoFailed && refetchRepo()
```

Comment:
Same here. It does the same thing, but a quick visual scan of this file and you'd miss that these were conditional invocations of functions. If you want brevity, then these calls might be well suited to the bracket-less syntax (though I do think using a block is better for making it obvious)
```suggestion
    if (clusterFailed) refetchCluster()
    if (appFailed) refetchApp()
    if (repoFailed) refetchRepo()
```